thundra/plugins/metric/metric_plugin.py

```diff
@@ -29,10 +29,10 @@ def __init__(self):
     def before_invocation(self, data):
         self.reporter = data['reporter']
         context = data['context']
-        event = data['event']
         stat_time = time.time() * 1000
```

Comment:
Can we use `datetime.datetime.now()` instead of `time.time()` for more accurate (milliseconds instead of seconds) time measurement?